spec/features/audit/report/allocation_spec.rb

```diff
@@ -1,31 +1,34 @@
 RSpec.feature "Content Allocation", type: :feature do
-  let!(:my_organisation) do
-    create(
-      :organisation,
-    )
-  end
-
-  let!(:me) do
-    create(
-      :user,
-      organisation: my_organisation,
-    )
+  scenario "Filter allocated content" do
+    given_i_am_auditing_a_content_item
+    then_i_see_all_content_items
+    given_that_i_filter_by_attributes_that_do_not_apply_to_any_content_items
```

Comment:
According to standard [Cucumber parlance](https://github.com/cucumber/cucumber/wiki/Given-When-Then), you should use `given` to "put the system in a known state before the user (or external system) starts interacting with the system".

You should use `when` here, as this step is here "to describe the key action the user performs".

As a rough rule of thumb, I'd only ever expect to see "Given" at the start of a test.

In general, this test is written a little vaguely, too. I'd probably expect a bit more detail about what exactly it is we're testing. For example, our scenario is filtering *allocated* content, but nowhere in your test do you mention the concept of allocation. I'd expect maybe:

> Given I have been allocated a content item
> When I visit the report page
> Then I can see the content item I have been allocated
> When I select "no one" in the "allocated to" filter
> Then I cannot see the content item I have been allocated